source/development/plugins.rst

```diff
@@ -146,27 +206,18 @@ Displaying a page from a Smarty template would look like:
    // display page
    $this->view->render(
       $response,
-      'file:[' . $module['route'] . ']file.tpl', [
+      'file:[' . $this->getModuleRoute() . ']file.tpl', [
           'require_dialog' => true,
           'list_values'    => $myvalues
       ]
    );
 
-The use of the ``$module['root']`` ensures the file you are trying to load is the one of your plugin. Without that, if Galette or another plugin provides a `file.tpl` file, it may be loaded frinstead of the one from your plugin, and this won't work.
-Then, ``file.tpl`` is core file ``file.tpl``, and ``file:[abcde]file.tpl`` the ``file.tpl`` file from plugin which identifier is ``abcde``.
+The use of the ``$this->getModuleRoute()`` ensures the file you are trying to load is the one of your plugin. Without that, if Galette or another plugin provides a `file.tpl` file, it may be loaded instead of the one from your plugin, and this won't work.
```

Comment:
"ensures the file you are trying to load is the one of your plugin"  -> "ensures the file you are trying to load is the one from your plugin"